app/src/main/java/com/example/plantsapp/presentation/ui/tasksfordays/TasksForDaysViewModel.kt

```diff
@@ -8,7 +9,21 @@ import java.util.Date
 import javax.inject.Inject
 
 @HiltViewModel
-class TasksForDaysViewModel @Inject constructor() : ViewModel() {
+class TasksForDaysViewModel @Inject constructor(
+    private val statisticsAppResolver: StatisticsAppResolver
+) : ViewModel() {
 
     val todayDate: LiveData<Date> = MutableLiveData(Date())
+
+    private val _isStatisticsAppInstalled: MutableLiveData<Boolean> = MutableLiveData(
+        statisticsAppResolver.isAppInstalled()
+    )
+    val isStatisticsAppInstalled: LiveData<Boolean> get() = _isStatisticsAppInstalled
```

Comment:
You can do
 `val isStatisticsAppInstalled: LiveData<Boolean>  = MutableLiveData(statisticsAppResolver.isAppInstalled())`